src/client/cli/formatter/table_formatter.cpp

```diff
@@ -58,42 +58,47 @@ std::string mp::TableFormatter::format(const InfoReply& reply) const
 {
     fmt::memory_buffer buf;
 
-    for (const auto& info : format::sorted(reply.info()))
+    for (const auto& info : format::sorted(reply.detailed_report().details()))
```

Comment:
Currently no update needed since the required message structure stayed the same (eg. `details()` is a repeated field where each item containing a subfield named `name`. This is still true after the rpc changes)

Reply:
Right, but once we start including snapshots they need to appear last. The relevant paragraph from the spec:

> Snapshot and instance arguments can be mixed. In that case, all requested instances will be displayed first, sorted alphabetically (as today), followed by all requested snapshots, sorted with the same criteria as in the snapshot overview. An MVP may refuse such mixing. The --all flag will now designate all instances and all snapshots.